tests.js

```diff
@@ -4327,7 +4327,7 @@ window.Specs = {
 					"tr": "#rubypos",
 					"dev": "#rubypos"
 				},
-				"tests": ["over", "under", "inter-character"]
```

Comment:
It looks like that change shouldn't be part of this patch.

Sebastian

Reply:
oups...